.github/workflows/e2e_tests.yaml

```diff
@@ -97,6 +97,9 @@ jobs:
         with:
           user-name: sdk-user
 
+      - name: Add kueue resources
+        run: kubectl apply --server-side -f "https://github.com/kubernetes-sigs/kueue/releases/download/v0.6.2/manifests.yaml"
```

Comment:
I think we should use the manifests from opendatahub-io/kueue

Reply:
I would consider SDK as upstream, so it should be integrated with upstream components.